### ISSUE TITLE
Fix race-condition in grunt-bower-task

### ISSUE THREAD

**Issue by peterblazejewicz** [OmniSharp/generator-aspnet#254]
By default `grunt-bower-task` will remove (clean) installed Bower package and try to copy it, this leads to race-condition:
```
grunt
Running "bower:install" (bower) task
>> Installed bower packages
Fatal error: ENOENT, no such file or directory 'wwwroot/lib/jquery-validation
```
The fix is to not copy already installed packages - as the same directory is used in `.bowerrc` config and in `grunt-bower-task` `targetDir` option.
Similar:
https://github.com/yatskevich/grunt-bower-task/issues/134

**Comment by peterblazejewicz** [comment]
as a comment: before `beta5` all client-side templates were included with source of template - so no Bower managed installation have been ocurring during installation. After `beta5`, all client side dependencies are installed via Bower (as part of build process or via Grunt task).

**Comment by peterblazejewicz** [comment]
closed via: 96e1c17
